pkg/Microsoft.ML.CpuMath/build/netstandard2.0/Microsoft.ML.CpuMath.props

```diff
@@ -0,0 +1,24 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Is the `xmlns` metadata needed?

Reply:
We put it in there to support older MSBuild's - specifically we still support VS 2015 if you have the netstandard2.0 stuff installed on it.

Reply:
We put it in there because this `.props` file gets loaded by the end-user's project. So we want to support as old of tools as possible. I believe we still support VS 2015 if you have the `netstandard2.0` support installed.

Reply:
Makes sense.